pkg/pillar/agentlog/agentlog.go

```diff
@@ -241,13 +251,47 @@ func listenDebug(log *base.LogObject, stacksDumpFileName, memDumpFileName string
 	}
 
 	info := `
-		This server exposes the net/http/pprof API.</br>
-		For examples on how to use it, see: <a href="https://pkg.go.dev/net/http/pprof">https://pkg.go.dev/net/http/pprof</a></br>
-	    <a href="debug/pprof/">pprof methods</a></br></br>
```

Comment:
Now you removed this link? I used that in the browser ...

Reply:
Returned the link